wagtail/admin/static_src/wagtailadmin/scss/userbar.scss

```
@@ -111,6 +111,21 @@ $positions: (
     margin: 0;
   }
 
+  .w-userbar-axe-count {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $color-button-no;
+    border-radius: 100px;
```

Won’t visually change anything but for consistency with other code where we make circles:

```suggestion
    border-radius: theme('borderRadius.full');
```